whois-api/src/main/java/net/ripe/db/whois/api/rest/DomainObjectService.java

```diff
@@ -63,14 +64,17 @@ public class DomainObjectService {
     private final WhoisObjectMapper whoisObjectMapper;
     private final LoggerContext loggerContext;
 
+    private final ServiceInputChecker servicesInputChecker;
+
     @Autowired
     public DomainObjectService(
             final WhoisObjectMapper whoisObjectMapper,
             final InternalUpdatePerformer updatePerformer,
-            final LoggerContext loggerContext) {
+            final LoggerContext loggerContext, final ServiceInputChecker serviceInputChecker) {
```

Comment:
Can you add a BusinessRuleValidator for this? Check any implementation of this interface.

Reply:
There is already a NServerValidator business rule. Can you have a look